### PR TITLE
Feature/isolation release

### PR DESCRIPTION
**Pull request by bgmoshe**

### CODE REVIEW

File: src/corona_hakab_model/detection_model/healthcare.py [hunk 1]
```diff
@@ -97,6 +104,14 @@ def progress_tests(self, new_tests: List[PendingTestResult]):
         for agent, test_result, _ in new_results:
             if test_result:
                 self.positive_detected_today.add(agent.index)
+                if self.manager.consecutive_negative_tests[agent.index] > 0:
```
Comment (meirhalachmi) [inline src/corona_hakab_model/detection_model/healthcare.py]: is this "if" really necessary? just set the value to 0

File: src/corona_hakab_model/generation/circles_consts.py [hunk 2]
```diff
@@ -138,6 +138,7 @@ def from_file(cls, param_path):
     def export(self, export_path, file_name: str = "circles_consts.json"):
         if not file_name.endswith(".json"):
             file_name += ".json"
+        jsonpickle.set_encoder_options('json', sort_keys=True, indent=4)
```
Comment (meirhalachmi) [inline src/corona_hakab_model/generation/circles_consts.py:139]: <3

**Review by meirhalachmi — APPROVED** [review]

File: src/corona_hakab_model/detection_model/healthcare.py (hunk 1 above)
Comment (bgmoshe) [inline src/corona_hakab_model/detection_model/healthcare.py]:
```suggestion
```

File: src/corona_hakab_model/detection_model/healthcare.py [hunk 3]
```diff
@@ -97,6 +104,14 @@ def progress_tests(self, new_tests: List[PendingTestResult]):
...
+                    self.manager.consecutive_negative_tests[agent.index] = 0
```
Comment (bgmoshe) [inline src/corona_hakab_model/detection_model/healthcare.py]:
```suggestion
                self.manager.consecutive_negative_tests[agent.index] = 0
```

**Pull request merged**